app/models/shipit/undeployed_commit.rb

```diff
@@ -31,5 +37,10 @@ def deploy_disallowed?
     def deploy_discouraged?
       stack.maximum_commits_per_deploy && index >= stack.maximum_commits_per_deploy
     end
+
+    def blocked?
+      return @blocked if defined?(@blocked)
+      @blocked = super
```

Comment:
`@blocked ||= super`? Or is this not possible on older Rubies?

Reply:
`||=` works since pretty much Ruby's inception.

However it's a shortcut for `foo = (defined?(foo) && foo) || something`.

So it can't be used to memoize `nil` and `false` values, which I need here.